src/App.tsx

```diff
@@ -98,7 +114,7 @@ export default () => {
 
   return (
     <div className="App">
-      <LanguageSelector name="English" />
+      <LanguageSelector name={langStore.getItem('lang') as string} />
```

Comment:
see my comment above about props.
We don't need to do this if the default is already picked up from langStore, and the opposite is also true.
I prefer removing the props but I let you decide.